README.md

```diff
@@ -66,17 +72,44 @@ In order to get started quickly, you can deploy the docker image to a DigitalOce
 
 *Disclaimer*: The DigitalOcean server is publicly accessible on the Internet. Do not put sensitive information on the network that you would not want someone else to know. Anyone with access to the network will be able to use the root account above.
 
-### Soroban RPC Server
+### Soroban Development
+
+For local development of smart contracts on Stellar using [Soroban], run a `standalone` network and the Soroban stack locally via the `stellar/quickstart:soroban-dev` image:
+
+```
+$ docker run --rm -it \
+    -p "8000:8000" \
+    --name stellar \
+    stellar/quickstart:soroban-dev \
+    --standalone \
+    --enable-soroban-rpc \
+    --protocol-version 20
```

Comment:
Why is protocol version 20 mentioned here? It shouldn't be required. The quickstart image will automatically upgrade the network to its maximum available network.

Reply:
it gave some visibility of the network level at use for soroban, removed it, convention over config,
https://github.com/stellar/quickstart/pull/373/commits/73c0b9fd87e1d82b3a55b5063ed1999d201a49e5